themes/ddbasic/sass/components/ding-react.scss

```diff
@@ -0,0 +1,95 @@
+// ding_react element styling.
+
+@import '../base.scss';
+
+.ddb-btn {
+  font-size: 14px;
```

Comment:
Sorry @kasperg, but why would you wanna define this specifically for this button?

Also why is this button in general not like the other buttons, using the a-tag with the .action-button class?
As an example, if you have color-defined your action-buttons to darkgreen and have increased your font for all body it would result in this:
![image](https://user-images.githubusercontent.com/287648/80100715-a0461400-8570-11ea-957a-e3b53d682426.png)